src/store/actions.js

```diff
@@ -15,17 +15,13 @@ export default {
 
   // lists
 
-  _fetchListsForUser({ commit, dispatch }) {
-    return this._vm.$axios.get(`${this._vm.$apiBasePath}lists`)
-      .then(response => {
-        commit('setLists', response.data);
-        dispatch('_setListIdFromLocalStorage');
-      })
-      .catch(error => {
-        throw error;
-      });
+  async _fetchListsForUser({ commit, dispatch }) {
+    const { data: responseLists } = await this._vm.$axios.get(`${this._vm.$apiBasePath}lists`);
+
+    commit('setLists', responseLists);
+    dispatch('_setListIdFromLocalStorage');
```

Comment:
what about return a promise here? Or how it works here?
Before we always return a Promise that in components we can use "then"-chaining.

Reply:
Ando also - it's needed to be checked everywhere where it changed to async

Reply:
Oh, as I read after, async function always return a promise, am I right?